src/pybind/mgr/dashboard/frontend/src/styles/defaults.scss

```diff
@@ -45,6 +46,8 @@ $color-light-gray: #d1d1d1 !default;
 $color-soft-gray: #ddd !default;
 $color-white-gray: #eee !default;
 $color-shade-gray: #efefef !default;
+$color-gray13: #212121;
+$color-gray20: #333333 !default;
```

Comment:
I just found out looking for a name in order to not name it myself  that https://www.color-hex.com/color/ shows gray13 and gray20 as the right color names for the colors. I first looked into another color naming system where both colors had the same name in the end, but I also asked myself would I have the slightest clue what color "Mine Shaft" or "Gray13" that would be? I bet not. Another thing to think about is that both colors are darker than our `dark-gray` color.

May be this is something to be discussed in another PR or an Issue. That said you don't have to find a solution for this in this PR. (At least from my side.)

Reply:
Yes, I initially named it mine-shaft but then I changed it.